README.md

```diff
@@ -76,7 +76,7 @@ Like Search-That-Hash? We have 2 other projects you'll **love**!
       <tr>
         <td>Instantly identify what type of hash it is. With support for over 3000 hashes, descriptions and popularity ratings!</td>
         <td>
-          Automatically decrypt ciphers without knowing the key, decode encodings, crack hashes, and more. Use this when you have some sort of encoded/encrypted text and you're not sure what it is. With support for over 50+ encryptions/encodings!
```

Comment:
You got rid of `text` so now it doesn't make sense. You need to add it back in or reword it.